src/status_im2/contexts/chat/messages/view.cljs

```diff
@@ -1,27 +1,27 @@
 (ns status-im2.contexts.chat.messages.view
```

Comment:
this namespace still reverted

Reply:
@flexsurfer reverted this file. As for the formatting, I keep running `make lint-fix` but I am not sure if it formatting the files correctly

Reply:
yeah i think it doesn't work, it didn't work for me as well you should run zprint for each file manually, what IDE do you use ?

Reply:
@yqrashawn is off today

Reply:
@flexsurfer I use IntelliJ

Reply:
we can't merge it as is, because its difficult to review, and also many PRs will be conflicting because of formatting

Reply:
@flexsurfer how to run zprint for each file manually?

Reply:
two ways 
1. add zprint as external tool and just run it in the file in IDE 
2. or run it un shell just run zprint with file path `zprint "{:search-config? true}" -fw src/status_im2/contexts/activity_center/events.cljs`

https://github.com/status-im/status-mobile/pull/14520#issue-1491418847

Reply:
@flexsurfer so I have run it manually on each of the changed files, but for some reason now 100s of files got reformatted

Reply:
yeah i had this issue as well in my PR, cc @yqrashawn

Reply:
The diff seems caused by running clojure-lsp clean-ns after zprint, I'll try on my side

Reply:
https://github.com/status-im/status-mobile/compare/develop...45231b190f6256940ec377c1f68ec6019f4f9fb1
This one is better, but still some, I guess, unrelated diff. Which one is last commit before trying to rebase against upstream, with that commit, I can get to the minimum diff

Reply:
@yqrashawn this is the last one before so many files got reformatted https://github.com/status-im/status-mobile/pull/14631/commits/68080de402ab0d7f218d4ecf804a3c4f70779a4f

But why is `clojure-lsp clean-ns` messing up the formatting. If it was already formatted correctly then it shouldn't change anything right?
Also currently when running `make lint-fix` is doesn't really fix all the formatting as shown is this other PR: https://github.com/status-im/status-mobile/pull/14635

Reply:
`clojure-lsp clean-ns` and `zprint` currently conflict with each other when formatting `ns` form, so it's required to run `zprint` after `clojure-lsp`
I'll create a pr to tweak `zprint`'s rule for `ns` forms

`make lint-fix` only applies to file staged (git add) in git `git diff --diff-filter=d --cached --name-only`

Reply:
> `make lint-fix` only applies to file staged (git add) in git `git diff --diff-filter=d --cached --name-only`

oh that's good to know

Reply:
Oh, I just found that this is not the old way. The old `make lint-fix` will fix everything in src. I'll fix this soon.